scripts/etc/init.d/rcS

```diff
@@ -760,7 +760,7 @@ cd /rootfs/boot/config/apt/
 # iterate through all the *.list files and add them to /etc/apt/sources.list.d
 for listfile in ./*.list
```

Comment:
https://github.com/debian-pi/raspbian-ua-netinst/pull/215#issuecomment-97498279 which points to http://www.dwheeler.com/essays/filenames-in-shell.html#globbing which explains why you should use `./*.list`